sdk/containerregistry/azure-containers-containerregistry/src/main/java/com/azure/containers/containerregistry/implementation/UtilsImpl.java

```diff
@@ -50,126 +55,30 @@
 /**
  * This is the utility class that includes helper methods used across our clients.
  */
-final class Utils {
-    private static final String CONTINUATIONLINK_HEADER_NAME;
-    private static final Pattern CONTINUATIONLINK_PATTERN;
+public final class UtilsImpl {
     private static final String CLIENT_NAME;
     private static final String CLIENT_VERSION;
     private static final int HTTP_STATUS_CODE_NOT_FOUND = 404;
     private static final int HTTP_STATUS_CODE_ACCEPTED = 202;
-    static final String CONTAINER_REGISTRY_TRACING_NAMESPACE_VALUE = "Microsoft.ContainerRegistry";
+    private static final String CONTINUATIONLINK_HEADER_NAME;
+    private static final Pattern CONTINUATIONLINK_PATTERN;
+
+    public static final String OCI_MANIFEST_MEDIA_TYPE;
+    public static final String DOCKER_DIGEST_HEADER_NAME;
+    public static final String CONTAINER_REGISTRY_TRACING_NAMESPACE_VALUE = "Microsoft.ContainerRegistry";
```

Comment:
Yes, they need to be public since they are getting used in a different package. 

I initially had a Utils and UtilsImpl but decided to combine them instead.